src/reducers/scopes.js

```diff
@@ -1,16 +1,21 @@
 import Immutable from "immutable";
 import { normalize } from "normalizr";
 import scopeSchema from "../schemas/scopes";
-import { GET_SCOPES_SUCCESS } from "../actions/scopes";
+import { GET_SCOPES_SUCCESS, GET_MY_SCOPE_SUCCESS } from "../actions/scopes";
 
-const initialState = Immutable.fromJS({});
+const initialState = Immutable.fromJS({ scopes: {}, defaultScope: null });
```

Comment:
Poor naming. The reducer gets included as "scopes", so the scope index here should be called "index" or suchlike. Alternatively, "defaultScope" belongs in settings, and not here, which means reverting to the previous structure would be better, with no unneeded nesting.

Reply:
Given that defaultScope seems to belong in settings instead, I'd say revert to previous.